facebook/OpenBIC
# fby3.5: cl:Support DPV2 sensor

Summary:
- Add DPV2 temperature/voltage/current/power sensor.
- Remove checking VR type debug message.

Test Plan:
- Build code: Pass
- Get DPV2 sensor: Pass
- Check debug message is removed in BIC console after BIC boot up: Pass

Log:
Get DPV2 sensor
root@bmc-oob:~# sensor-util slot1 --threshold | grep "DPV2"
DPV2_2_12V_Vin               (0x91) :   12.40 Volts | (ok) | UCR: 13.19 | UNC: 12.94 | UNR: 14.04 | LCR: 10.79 | LNC: 11.05 | LNR: 10.01
DPV2_2_12V_Vout              (0x92) :   12.38 Volts | (ok) | UCR: 13.19 | UNC: 12.94 | UNR: 14.04 | LCR: 10.79 | LNC: 11.05 | LNR: 10.01
DPV2_2_12V_Iout              (0x93) :    0.12 Amps  | (ok) | UCR: 15.49 | UNC: 15.14 | UNR: 18.88 | LCR: NA | LNC: NA | LNR: NA
DPV2_2_EFUSETemp             (0x94) :   22.38 C     | (ok) | UCR: 100.00 | UNC: NA | UNR: 150.00 | LCR: NA | LNC: NA | LNR: NA
DPV2_2_EFUSEPWR              (0x95) :    1.18 Watts | (ok) | UCR: 186.00 | UNC: NA | UNR: 227.00 | LCR: NA | LNC: NA | LNR: NA

Before removing VR type debug message
[BIC console]
Hello, welcome to yv35 craterlake 2022.19.1
BIC class type(class-1), 1ou present status(0), 2ou present status(1), board revision(0x2)
Failed to read VR register(0xad)
ected ep[VR type: ISL69259
0x2] as VR type: ISL69259
IN endpoiVR type: ISL69259
,000] VR type: ISL69259
<inf>VR type: ISL69259
 usb_dc_aVR type: ISL69259
speed: seVR type: ISL69259
lect ep[0VR type: ISL69259
x3] as OUVR type: ISL69259
T endpoinVR type: ISL69259
[0VR type: ISL69259
0:00:00.0VR type: ISL69259
56,000] R type: ISL69259
[0m<inf> VR type: ISL69259
kcs_aspeeVR type: ISL69259
d: KCS3: VR type: ISL69259
addr=0xcaVR type: ISL69259
ipmi_init
[set_DC_status] gpio number(15) status(1)
[set_post_status] gpio number(1) status(0)

After removing VR type debug message
[BIC console]
Hello, welcome to yv35 craterlake 2022.19.1
BIC class type(class-1), 1ou present status(0), 2ou present status(1), board revision(0x2)
pre-selected ep[0x2] as IN endpoint
[00:00:00.001,000] <inf> usb_dc_aspeed: select ep[0x3] as OUT endpoint
[00:00:00.056,000] <inf> kcs_aspeed: KCS3: addr=0xca2, idr=0x2c, odr=0x38,ipmi_init
[set_DC_status] gpio number(15) status(1)
[set_post_status] gpio number(1) status(1)
